bindings/experimental/distrdf/python/DistRDF/Backends/Base.py

```diff
@@ -159,7 +159,16 @@ def mapper(current_range):
                 list: This respresents the list of (mergeable)values of all
                 action nodes in the computational graph.
             """
-            import ROOT
+            # - Disable graphics functionality in ROOT. It is not needed inside
```

Comment:
Perhaps I would place the comment for each code line separately, instead of the two comments together?